/scratch/work/uotilav2/quantum-portfolio/venv/lib64/python3.9/site-packages/cma/s.py:15: UserWarning: Could not import matplotlib.pyplot, therefore ``cma.plot()`` etc. is not available
  _warnings.warn('Could not import matplotlib.pyplot, therefore'
/scratch/work/uotilav2/quantum-portfolio/venv/lib64/python3.9/site-packages/networkx/utils/backends.py:135: RuntimeWarning: networkx backend defined more than once: nx-loopback
  backends.update(_get_backends("networkx.backends"))
[                       0%                       ][************          25%                       ]  2 of 8 completed[******************    38%                       ]  3 of 8 completed[**********************50%                       ]  4 of 8 completed[**********************62%*****                  ]  5 of 8 completed[**********************75%***********            ]  6 of 8 completed[**********************88%*****************      ]  7 of 8 completed[*********************100%***********************]  8 of 8 completed
[                       0%                       ][                       0%                       ][                       0%                       ][                       0%                       ][**********************62%*****                  ]  5 of 8 completed[**********************75%***********            ]  6 of 8 completed[**********************88%*****************      ]  7 of 8 completed[*********************100%***********************]  8 of 8 completed
[                       0%                       ][                       0%                       ][                       0%                       ][********************* 44%                       ]  4 of 9 completed[**********************56%**                     ]  5 of 9 completed[**********************56%**                     ]  5 of 9 completed[**********************78%************           ]  7 of 9 completed[**********************89%******************     ]  8 of 9 completed[*********************100%***********************]  9 of 9 completed
[                       0%                       ][                       0%                       ][                       0%                       ][**********************56%**                     ]  5 of 9 completed[**********************67%*******                ]  6 of 9 completed[**********************89%******************     ]  8 of 9 completed[**********************67%*******                ]  6 of 9 completed[                       0%                       ][*********************100%***********************]  9 of 9 completed
/var/spool/slurmd/job6255356/slurm_script: line 34: 39337 Killed                  python experiments.py ${SLURM_ARRAY_TASK_ID} ${TOTAL_BATCHES}
slurmstepd: error: Detected 1 oom_kill event in StepId=6255356.batch. Some of the step tasks have been OOM Killed.
